react-app/src/ExerciseOverview/index.js

```diff
@@ -3,16 +3,20 @@ import React, { useState } from 'react'
 import Header from '../Utilities/Header'
 import Summary from './Summary'
 import ExerciseCard from './ExerciseCard'
+import { CORE_EXERCISE_DATA } from '../../data/CoreExerciseData'
 import { STRETCHING_EXERCISE_DATA } from '../../data/StretchingExerciseData'
 import PageTemplate from '../Utilities/PageTemplate'
 import ExerciseDetail from './ExerciseDetail'
 import ThemeColor from '../Utilities/ThemeColor'
 
-export default function ExerciseOverview() {
-  const exerciseData = STRETCHING_EXERCISE_DATA
-  const totalSteps = exerciseData.length
+export default function ExerciseOverview({ route }) {
+  const { id, title } = route.params
   const [showDetail, setShowDetail] = React.useState(false)
   const [selectedStep, setSelectedStep] = useState(1)
+  const exercises = { 1: STRETCHING_EXERCISE_DATA, 2: CORE_EXERCISE_DATA }
+  const exercise = exercises[id]
+  console.log(id, title)
+  const duration = Math.ceil(exercise.reduce((acc, curr) => acc + curr.duration, 0) / 60)
```

Comment:
Can we use full description as variable name here like accumulation, currentMoveDuration to improve readability?

Reply:
Of course! I changed it